apple/Tests/TestCases/RET/ManifestBuilding/ManifestBuildingTests.swift

```diff
@@ -9,9 +9,9 @@ final class ManifestBuildingTests: XCTestCase {
 			addressOfReceivingAccount: AccountAddress.sample
 		)
 		
-		XCTAssert(manifest.description.contains("CALL_METHOD"))
-		XCTAssert(manifest.description.contains(AccountAddress.sample.description))
-		XCTAssert(manifest.description.contains("lock_fee"))
+		XCTAssert(manifest.instructionsString.contains("CALL_METHOD"))
```

Comment:
can you revert this please, I've added many more in my branch and I wanna keep `.description` for sake of consistency :)